subsys/bluetooth/mesh/settings.c

```diff
@@ -1424,17 +1661,137 @@ static void store_pending_nodes(void)
 	}
 }
 
-static struct node_update *node_update_find(u16_t addr,
-					    struct node_update **free_slot)
+static void store_cdb_subnet(const struct bt_mesh_cdb_subnet *sub)
+{
+	struct net_key_val key;
+	char path[20];
```

Comment:
This path is too short for a null terminator. Should also probably account for more than 9 net indexes.

This is the same for all path strings in these `store` and `clear` functions

Reply:
Good catch :+1: